src/libraries/System.Private.Xml/src/System/Xml/Core/XmlParserContext.cs

```diff
@@ -185,7 +186,7 @@ public XmlSpace XmlSpace
             }
```

Comment:
Cannot comment on the row, properties `XmlLang, InternalSubset, SystemId, BaseURI, PublicId, DocTypeName` could have [AllowNull] attribute, but might not intentional

Reply:
good point on `[AllowNull]`. I'll go with whatever state it is in right now